tests/integration/sync/test_sync_infra.py

```diff
@@ -315,3 +315,50 @@ def test_cdk_templates(self, template_file, template_after, function_id, depende
                 lambda_response = json.loads(self._get_lambda_response(lambda_function))
                 self.assertIn("extra_message", lambda_response)
                 self.assertEqual(lambda_response.get("message"), "9")
+
+
+@skipIf(SKIP_SYNC_TESTS, "Skip sync tests in CI/CD only")
+@parameterized_class([{"dependency_layer": True}, {"dependency_layer": False}])
+class TestSyncInfraWithJava(SyncIntegBase):
+    dependency_layer = True
```

Comment:
Should we remove this initialization? It is already being set by the parameterized class.

Reply:
Good catch, I kept it for testing without parameters, let me remove it 👍🏼